docs/specs/#2072 - Add some PowerToys utilities/spec.md

```diff
@@ -0,0 +1,165 @@
+---
+author: Kayla Cinnamon @cinnamon-msft
+created on: 2023-12-22
+last updated: 2024-01-04
+issue id: #2072
+---
+
+# Add some PowerToys utilities
+
+## 1. Overview
+
+### 1.1 Establish the Problem
+
+Dev Home is intended to be a centralized location for developers to help them stay productive on Windows. One feature we had been thinking of is adding helpful utilities into Dev Home for quick and easy access.
+
+### 1.2 Introduce the Solution
+
+PowerToys has created a few utilities with the original intent of migrating them into Dev Home. These utilities include Registry Preview, Hosts File Editor, and Environment Variables. We can add these utilities into Dev Home as the "stable" versions of these tools, while maintaining their "preview" versions in PowerToys. By adding these utilities to Dev Home, we can provide developers with more value out of the box in Windows while also provide more awareness to PowerToys.
+
+### 1.3 Rough-in Designs
+
+![V1 of PowerToys in Dev Home](./v1-pt-in-dev-home.png)
+
+## 2. Goals & User Cans
+
+### 2.1 Goals
+
+1. Add additional functionality to Dev Home
+2. Provide helpful inbox utilities to the user
+3. Bring awareness to PowerToys
+4. Provide stable versions of PowerToys utilities
+5. Provide a surface within Dev Home for additional utilities outside of PowerToys
+
+### 2.2 Non-Goals
+
+1. This feature is not intended to replace PowerToys
+2. This feature is not intended to deprecate existing PowerToys
+
+### 2.3 User Cans Summary Table
+
+| No. | User Can | Pri |
```

Comment:
1. do we need to have some type of linking system back to PowerToys if they want the PowerToys version of the app? 
2. Does dev home need a way to direct launch the PT version of it vs the Dev home if it is installed